crates/proto/src/rr/rdata/svcb.rs

```diff
@@ -1051,6 +1054,7 @@ impl fmt::Display for Unknown {
 /// ```
 pub fn read(decoder: &mut BinDecoder<'_>, rdata_length: Restrict<u16>) -> ProtoResult<SVCB> {
     let start_index = decoder.index();
+    println!("{:?}", start_index);
```

Comment:
these prtinln's look like debug statements?

Reply:
Oops, sorry about that.